app/inputsearchmodel.h

```diff
@@ -0,0 +1,36 @@
+/***************************************************************************
```

Comment:
I do not know, all logic is in `MerginProjectModel` anyways, so I still do not understand why we need new class just as a storage of one string. 

Can you just remove input search model class alltogether?